drivers/sensor/pmw3360/pmw3360.c

```diff
@@ -628,7 +644,20 @@ static void irq_handler(struct device *gpiob, struct gpio_callback *cb,
 		.type = SENSOR_TRIG_DATA_READY,
 		.chan = SENSOR_CHAN_ALL,
 	};
+
 	handler(DEVICE_GET(pmw3360), &trig);
+
+	key = k_spin_lock(&pmw3360_data.lock);
+	if (pmw3360_data.data_ready_handler) {
+		err = gpio_pin_enable_callback(pmw3360_data.irq_gpio_dev,
+					       PMW3360_IRQ_GPIO_PIN);
+	}
+	k_spin_unlock(&pmw3360_data.lock, key);
+
+	if (unlikely(err)) {
+		LOG_ERR("Cannot disable IRQ");
```

Comment:
"Cannot re-enable" (same story in `paw3212.c`).

Reply:
Fixed